admin/server/github.go

```diff
@@ -21,6 +21,8 @@ func (s *Server) GetGithubRepoStatus(ctx context.Context, req *adminv1.GetGithub
 	if claims.OwnerType() != auth.OwnerTypeUser {
 		return nil, status.Error(codes.Unauthenticated, "not authenticated")
 	}
+	// TODO need a project ID here to check permissions
```

Comment:
This is not needed since this is actually a check for whether the calling user has granted github access to a specific repo, it's not related to a specific project (they grant access before the project is created, so it might not exist yet).